controllers/cluster_controller.go

```diff
@@ -321,9 +326,11 @@ func (r *ClusterReconciler) reconcileControlPlaneInitialized(ctx context.Context
 // controlPlaneMachineToCluster is a handler.ToRequestsFunc to be used to enqueue requests for reconciliation
 // for Cluster to update its status.controlPlaneInitialized field
 func (r *ClusterReconciler) controlPlaneMachineToCluster(o handler.MapObject) []ctrl.Request {
+	logger := r.Log
```

Comment:
I'd undo the changes in this function unless you're planning on defining `WithValues` somewhere